.github/workflows/integration.yml

```diff
@@ -157,3 +164,10 @@ jobs:
           TEST_TARGET: ${{ matrix.target }}
           TERRAFORM_CLOUD_TOKEN: ${{ secrets.TERRAFORM_CLOUD_TOKEN }}
           NODE_OPTIONS: "--max-old-space-size=7168"
+
+  windows_integration_done:
```

Comment:
nit: We could combine this in a single "integration_success" job and depend on both linux and windows. No hard feelings either in favor or against it, though 😅

Reply:
@DanielMSchmidt any comment on why two is better than one? 😁